app/views/comments/settings.html.erb

```diff
@@ -1,28 +1,6 @@
 <% title "Comment Settings" %>
 
 <style>
-  .pill {
-    padding: 8px 20px;
-    border-radius: 100px;
-    color: white;
-    margin-right: 5px;
-    font-size: 14px;
-    border: 2px solid transparent;
-    margin-top: 6px;
-    display: inline-block;
-  }
-
-  .mute {
-    background-color: #EE7453;
-    color: #ffffff;
-  }
-
-  .unmute {
-    /* inverted background color */
-    background-color: #1395b8;
-    color: #ffffff;
-  }
-
   .container {
     text-align: center;
     padding: 30px 0px;
```

Comment:
I could probably just get rid of this entire inline style in this document.

Reply:
Yea, I don't think so - it will look off until you apply a bit more crayons there :)